pelican_bib/bib.py

```diff
@@ -7,49 +7,81 @@
 A Pelican plugin that populates the context with a list of formatted
 citations, loaded from a BibTeX file at a configurable path.
 
-The use case for now is to generate a ``Publications'' page for academic
+The use case for now is to generate a "Publications" page for academic
 websites.
+
+Configuration
+-------------
+
+``generator.settings['PUBLICATIONS_SRC']``:
+    Local path to the BibTeX file to read.
+    Each generator contains this list of publications.
+
+``generator.settings['PUBLICATIONS_SPLIT_BY']``:
+    The name of the bibtex field used for splitting the publications.
+    No splitting if title is not provided.
+
+``generator.settings['PUBLICATIONS_UNTAGGED_TITLE']``:
+    The title of the header for all untagged entries.
+    No such list if title is not provided.
+
+``generator.settings['PUBLICATIONS_DECORATE_HTML']``:
+    If set to True, elements of a publication entry (e.g. names, title)
+    will be decorated with a <span> tag with a specific class name
+
+``generator.settings['PUBLICATIONS_DEFAULT_TEMPLATE']``:
```

Comment:
This one got me thinking. I think it is cool to use a default template, but users might think that also applies for the "page version" of the plugin. In general, the two version are a bit entangled now (see comments before), and I am not sure how we can clean that up nicely. In this case, maybe we should make it clear we refer to the DIRECTIVE rather than a full page template.  This way we can have `PUBLICATIONS_DEFAULT_DIRECTIVE_TEMPLATE` and `PUBLICATIONS_DEFAULT_PAGE_TEMPLATE`. What do you think?

Reply:
Oh, this one isn't mentioned in the README. Thanks for the hint.

Sure, we can rename it to `PUBLICATIONS_DEFAULT_DIRECTIVE_TEMPLATE`.
But having a default template for the PAGE version does not make any sense, does it? The PAGE version works because you specify a particular template for your page (which then renders the bibliography).